__tests__/app.test.js

```diff
@@ -358,3 +358,25 @@ describe("GET /api/users", () => {
       });
   });
 });
+
+describe("200: GET /api/articles?topic=[input] should respond with all articles with a topic matching the query", () => {
+  test("200: GET /api/articles?topic=cats should respond with all articles with a topic of cats", () => {
+    return request(app)
+      .get("/api/articles?topic=mitch")
+      .expect(200)
+      .then(({ body }) => {
+        console.log(body)
+        body.articles.forEach((article) => {
+          expect(article.topic).toBe("mitch");
+        });
+      });
+  });
+  test("404: GET /api/articles?topic=[non-existent topic] should respond with Not Found if passed a topic that doesn't exist", () => {
+    return request(app)
+      .get("/api/articles?topic=pasta")
+      .expect(404)
+      .then(({ body }) => {
+        expect(body.msg).toBe("Not Found");
+      });
+  });
+});
```

Comment:
Really good testing so far, but there is one final test that should be added here.

What happens when a user makes a request to a valid topic, but one that his no associated articles? 
Hint: topic: "paper" has no associated articles